Reorganization of Jupyter content (#638)
ProjectPythia/pythia-foundations: foundations/terminal.md

## Overview

> You'd like to learn to run Python in the terminal. You also want to jump right in with some installation quick-start guidance before reading through our detailed chapter on [Installing and Managing Python with Conda](conda.md). This chapter is for you!

Reviewer: ```suggestion
If you'd like to learn to run Python in the terminal, this chapter is for you!
```

Reviewer: Just thought this was a little confusing.

Author: I changed the wording in [that overview](https://foundations.projectpythia.org/_preview/638/foundations/terminal/#overview) and also a similar overview for the [Python in Jupyter](https://foundations.projectpythia.org/_preview/638/foundations/jupyter/#overview) chapter. Hopefully clearer now. I'm just trying to clarify that both of these chapters are essentially "Quickstart" guides for the impatient, and more details are coming in later chapters.
